### PR TITLE
Add limitations section to SMS testing page to inform about montly SM…

### PR DESCRIPTION
**Pull request by yourtallness**
…S limits on development

### CODE REVIEW

File: docs/testing/test-emails-and-phones.mdx [hunk 1]
```diff
@@ -9,6 +9,21 @@ Most of Clerk's sign-in and sign-up flows involve verifying ownership of an emai
 
 Verification messages are used during sign-up, sign-in, and when adding an email address or phone number to an existing account.
 
+## Limitations
+
+If Clerk is used to deliver SMS for your development instance, a maximum of 20 SMS can be delivered per calendar month.
+
+After that, requests resulting in OTP SMS messages will be rejected.
+Other SMS notifications will still produce a webhook but not actually be sent to the target number.
```
Comment (S3Prototype) [inline docs/testing/test-emails-and-phones.mdx:17]:
```suggestion
Other SMS notifications will still produce a webhook but won't be sent to the target number.
```

File: docs/testing/test-emails-and-phones.mdx [hunk 2]
```diff
@@ -9,6 +9,21 @@ Most of Clerk's sign-in and sign-up flows involve verifying ownership of an emai
...
+
+The following cases does not count toward the limit:
+* SMS sent to US numbers
+* SMS sent to test numbers
+* Self-delivered SMS (i.e. not delivered by Clerk)
+* SMS for apps with a paid subscription
```
Comment (S3Prototype) [inline docs/testing/test-emails-and-phones.mdx:23]:
```suggestion
* SMS messages sent to US numbers
* SMS messages sent to test numbers
* Self-delivered SMS messages (i.e. not delivered by Clerk)
* SMS messages for apps with a paid subscription
```

**Review by S3Prototype — APPROVED** [review]
Approved with suggestions. Thanks for contributing!

File: docs/testing/test-emails-and-phones.mdx (hunk 2 above)
Comment (yourtallness) [reply docs/testing/test-emails-and-phones.mdx]: Done

File: docs/testing/test-emails-and-phones.mdx (hunk 1 above)
Comment (yourtallness) [reply docs/testing/test-emails-and-phones.mdx]: Done

File: docs/testing/test-emails-and-phones.mdx [hunk 3]
```diff
@@ -9,6 +9,21 @@ Most of Clerk's sign-in and sign-up flows involve verifying ownership of an emai
 
 Verification messages are used during sign-up, sign-in, and when adding an email address or phone number to an existing account.
 
+## Limitations
+
+If Clerk is used to deliver SMS for your development instance, a maximum of 20 SMS can be delivered per calendar month.
```
[the remark this answers is not in the document]
Comment (yourtallness) [reply docs/testing/test-emails-and-phones.mdx]: Done

**Pull request merged**